src/main/java/com/contentful/java/cda/Platform.java

```diff
@@ -114,7 +148,22 @@ private static Platform findPlatform() {
      * @return the version number of the android os, if set. Otherwise null.
      */
     @Override public String version() {
-      return tryGettingAndroidReleaseVersionString();
+      return versionName;
+    }
+
+//BEGIN TO LONG CODE LINES
+    /**
+     * Should a custom TLSSocketFactory enable TLS12?
+     * <p>
```

Comment:
close the paragraph or remove it

Reply:
Nope, it's called javadoc. no closing neccessary.